tools/exporter/README.md

```diff
@@ -1,74 +1,99 @@
-# Audit exporter tool
-
+# Audit Exporter Tool
 Allows you to export audit data from the SafetyCulture Platform and save them anywhere on your computer.
+Supported export formats: PDF, MS WORD (docx), JSON, and CSV. Media and Web Report Link exporting is also supported.
 
-Supported export formats: PDF, MS WORD (docx), JSON, and CSV. Media exporting is also supported.
+## Installation  
+``` 
+pip install safetyculture-sdk-python
+```
 
-## Installation
+This will install
+* SafetyCulture Python SDK -- See top level [README.md](https://github.com/SafetyCulture/safetyculture-sdk-python/blob/master/README.md) for more information. 
+* iAuditor Exporter Tool
+* README files
 
-  1. First install the Python SDK (see top-level Readme)
-  2. Switch to this directory (safetyculture-sdk-python/tools/exporter)
-  3. Execute the following command from the command line:
-    ``pip install -r requirements.txt``
-  4. Edit config.yaml and replace ``YOUR_SAFETYCULTURE_API_TOKEN`` with your SafetyCulture API token
 
+## Initial Setup
+IMPORTANT: If you have used previous versions of the exporter tool, you should run it from the same folder you have run the tool from in the past.
+Otherwise, the exporter will start exporting from the earliest available audits, rather than from where the last successful export left off. 
+The export tool reads and writes to a file named `last_successful.txt` to keep track of what has already been exported. You'll find this file anywhere that you have run the exporter tool before. 
+Alternatively, you can move the `last_successful.txt` file if you prefer to export from a different location.
 
-## How to run
+If this is your first time using the exporter tool, follow these steps to get set up: 
+1. To automatically create a configuration file (which is needed to run the exporter tool), run:  
+```
+iauditor_exporter --setup
+```
+* You will be prompted for an iAuditor username and password which will be used to generate an API token. 
+Note that your credentials will not be saved in any capacity.
+* A basic configuration file will be auto-generated. A configuration file is necessary to run the Exporter script.
+The file will be named `config.yaml` and be placed in a folder named `iAuditor Audit Exports` which will be created in your current directory. 
+2. Navigate into the `iauditor_exports_folder` folder just created:
+```
+cd 'iauditor_exports_folder'
+```
+3. To start exporting audits in PDF format, run the following command 
+```
+iauditor_exporter
+```
+#### Windows Users
+The location of `iauditor_exporter.exe` must be included in the system PATH variable in order to execute from the command line without including the full path to the .exe file.  
+Find the location of `iauditor_exporter.exe` by running 
+```
+> where iauditor_exporter
+```
+Add this path to the system PATH variable. 
+
+Directions on how to update Windows system PATH variable [here](https://www.computerhope.com/issues/ch000549.htm)
```

Comment:
I suggest to not include any links to third party sites in this README. Updating the system PATH should be easy to find using Google at least.